.github/workflows/pull-request.yml

```diff
@@ -123,4 +123,81 @@ jobs:
         if: env.RUN_JSDOCS == 'True'
         run: echo "Run JSdocs :${{ env.RUN_JSDOCS }}"
 
-          
\ No newline at end of file
+  Dev-Server:
+    name: Dev-Server
+    runs-on: ubuntu-latest
+    needs: Test-Application
+    strategy:
+      matrix:
+        node-version: [14.x]
+    services:
+      mongo:
+        image: mongo:4.4
+        options: >-
+          --health-cmd mongo
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 10
+        ports:
+          - 27017:27017
+      redis:
+        image: redis:6.0
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 10
+        ports:
+          - 6379:6379
+
+    env:
+      ACCESS_TOKEN_SECRET: ${{ secrets.ACCESS_TOKEN_SECRET }}
+      REFRESH_TOKEN_SECRET: ${{ secrets.REFRESH_TOKEN_SECRET }}
+      MONGO_DB_URL: "mongodb://localhost:27017/talawa-test-db"
+      RECAPTCHA_SITE_KEY: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      RECAPTCHA_SECRET_KEY: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      MAIL_USERNAME: "abc@gmail.com"
+      MAIL_PASSWORD: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      IS_SMTP: ""
+      SMTP_HOST: ""
+      SMTP_PASSWORD: ""
+      SMTP_USERNAME: ""
+      SMTP_PORT: ""
+      SMTP_SSL_TLS: ""
+      apiKey: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      appId: "1:XXXXXXXXXXXX:android:XXXXXXXXXXXXXXXXXXXXXX"
+      messagingSenderId: "XXXXXXXXXXXX"
+      projectId: "appname-XXXXXX"
+      storageBucket: "appname-XXXXXX.appspot.com"
+      iOSapiKey: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      iOSappId: "1:XXXXXXXXXXXX:ios:XXXXXXXXXXXXXXXXXXXXXX"
+      iOSmessagingSenderId: "XXXXXXXXXXXX"
+      iOSprojectId: "appame-XXXXXX"
+      iOSstorageBucket: "appame-XXXXXX.appspot.com"
+      iOSClientId: "XXXXXXXXXXXX-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.apps.googleusercontent.com"
+      iOSBundleId: "com.appname.app"
+      LAST_RESORT_SUPERADMIN_EMAIL: "abc@gmail.com"
+      COLORIZE_LOGS: "true"
+      LOG_LEVEL: "info"
+      REDIS_HOST: localhost
+      REDIS_PORT: 6379
+      NODE_ENV: production
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run dev server
+        run: npm run dev 
+
+      - name: Check health endpoint
+        run: bash ${GITHUB_WORKSPACE}/scripts/endpoint_health.sh
```

Comment:
Does the **${GITHUB_WORKSPACE}** variable point to **/** or **/.github**?

Reply:
.github/